### PR TITLE
Sccs for auth 4

### PR DESCRIPTION
**Pull request by maskpa100**

### CODE REVIEW

**Review by FightersForJustice — APPROVED** [review]

**Review by Viarcinski — APPROVED** [review]
Looks good to me

**Review by TURP1 — APPROVED** [review]

**Review by irinrudko — COMMENTED** [review]
let's use 'style' instead of 's'

**Review by irinrudko — COMMENTED** [review]
let's use 'button' word, not 'btn'. it doesn't save much space, but it's more difficult to read/write the code. I suggest 'button-primary'. etc/

File: styles/_mixin.scss [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    line-height: 24px;
+    cursor: pointer;
+  }
+@mixin Secondary-btn {
+    cursor: pointer;
+  }
+@mixin Outline-btn {
+  cursor: pointer;
+}
+@mixin Large {
```
Comment (irinrudko) [inline styles/_mixin.scss:18]: what is 'large'? how to use it? where to use it? not clear.
https://sass-lang.com/documentation/at-rules/mixin/

File: styles/_mixin.scss [hunk 2]
```diff
@@ -0,0 +1,98 @@
+
+
+@mixin Primary-btn {
```
Comment (irinrudko) [inline styles/_mixin.scss:3]: let's use 'button' word, not 'btn'. it doesn't save much space, but it's more difficult to read/write the code. I suggest 'button-primary'. etc

**Review by irinrudko — CHANGES_REQUESTED** [review]
please rename some of the things

**Review by irinrudko — CHANGES_REQUESTED** [review]
please rename some of the things

**Review by ilich01 — APPROVED** [review]

**Pull request merged**